src/components/unapprovedImagesTable/index.tsx

```diff
@@ -55,34 +135,116 @@ function responseToTableData(
     dateSubmitted: data.dateSubmitted,
     species: data.commonName,
     neighborhood: NEIGHBORHOOD_IDS[data.neighborhoodId],
+    submittedBy: data.uploaderName,
+    address: data.address,
   };
 }
 
 const UnapprovedImagesTable: React.FC<UnapprovedImagesTable> = ({
   fetchData,
   setSelectedImageIds,
+  useGridView,
+  approvedOrRejectedImageIds,
+  setApprovedOrRejectedImageIds,
 }) => {
+  const filteredData = useMemo(
+    () =>
+      fetchData
+        ? fetchData.filter(
+            (data) => !approvedOrRejectedImageIds.includes(data.imageId),
+          )
+        : [],
+    [fetchData, approvedOrRejectedImageIds],
+  );
+
   const tableData = useMemo(
-    () => (fetchData ? fetchData.map(responseToTableData) : []),
-    [fetchData],
+    () => (filteredData ? filteredData.map(responseToTableData) : []),
+    [filteredData],
```

Comment:
❓ What's the reason for putting `filteredData` into its own `useMemo` here? Seems like we only use it to compute `tableData` unless I'm missing something